presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/ValuesMatcher.java

```diff
@@ -74,10 +75,14 @@ public MatchResult detailMatches(PlanNode node, PlanNodeCost planNodeCost, Sessi
     @Override
     public String toString()
     {
-        return toStringHelper(this)
-                .add("outputSymbolAliases", outputSymbolAliases)
-                .add("expectedOutputSymbolCount", expectedOutputSymbolCount)
-                .add("expectedRows", expectedRows)
-                .toString();
+        ToStringHelper toStringHelper = toStringHelper(this)
+                .add("outputSymbolAliases", this.outputSymbolAliases);
+        if (expectedOutputSymbolCount.isPresent()) {
```

Comment:
you can do this fluently using `ToStringHelper.omitNulls()`. I think readability is then much better.